src/UITests/UILibrary/LiveMode.cs

```diff
@@ -1,17 +1,46 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using AccessibilityInsights.SharedUx.Properties;
 using OpenQA.Selenium.Appium.Windows;
+using UITests.Utilities;
 
 namespace UITests.UILibrary
 {
     public class LiveMode
     {
         WindowsDriver<WindowsElement> Session;
 
+        const string OpenFileToolBarAutomationID = "1001";
+        const string OpenFileFolderTextBoxAutomationID = "41477";
+        const string OpenFileFileTextBoxAutomationID = "1148";
+
         public LiveMode(WindowsDriver<WindowsElement> session)
         {
             Session = session;
         }
-        public bool OpenFile(string folder, string fileName) => false;
+
+        public bool OpenFile(string folder, string fileName)
+        {
+            if (!Session.FindAndClickByAutomationID(AutomationIDs.MainWinLoadButton)) return false;
+
+            if (!Session.FindAndClickByAutomationID(OpenFileToolBarAutomationID)) return false;
+
+            try
+            {
+                var folderTextbox = Session.FindElementByAccessibilityId(OpenFileFolderTextBoxAutomationID);
+                folderTextbox.SendKeys(folder);
+                folderTextbox.SendKeys(OpenQA.Selenium.Keys.Enter);
+
+                var fileTextbox = Session.FindElementByAccessibilityId(OpenFileFileTextBoxAutomationID);
+                fileTextbox.SendKeys(fileName);
+                fileTextbox.SendKeys(OpenQA.Selenium.Keys.Enter);
+
+                return true;
+            }
+            catch
```

Comment:
are there examples of what this is guarding against?

Reply:
See my other comment about the bool returns on the navigation methods. It's not really guarding against something so much as changing exceptions to a bool "has succeeded?" return, but I'm just going to get rid of these and let the methods throw exceptions per @DaveTryon's suggestion.